aries-site/src/pages/templates/forms.mdx

```diff
@@ -221,3 +194,32 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
 >
   <CustomizeExample />
 </Example>
+
+## Accessibility
+
+All Form input elements need to be contained within a FormField.
+
+**FormFields must include:** 
+
+- A "for" property that matches the id of its child input. If using Grommet, this is applied with the htmlFor prop.
+- A "name" property applied on both the FormField and its child input element
+- A clear, concise label
+
+## Exceptions and how to address them
+
+- A login/registration form, which typically shows two to three required fields do not need to be annotated in any way - the form is short and the necessity of completing the fields is relatively obvious.
+
+- For login/registration forms, apply `required={{ indicator: false }}` to the FormFields to remove the required indicator.
+
+- Forms with minimal fields are optimal but occasionally, capturing extra information would be highly beneficial for the user. The following are some options for addressing these cases: 
+
+- Create the form the optimal way: with only the minimum required fields so that the user can focus on completing their task. Separately, in a preferences or settings dashboard, for example, create those fields where the user can proactively add this helpful information at another time. 
```

Comment:
```suggestion
   - Create the form the optimal way: with only the minimum required fields so that the user can focus on completing their task. Separately, in a preferences or settings dashboard, for example, create those fields where the user can proactively add this helpful information at another time. 
```
This feels like it should be a sub-bullet of the bullet above